src/main/kotlin/sidechain/eth/util/DeployHelper.kt

```diff
@@ -65,16 +65,16 @@ class DeployHelper(ethereumConfig: EthereumConfig, ethereumPasswords: EthereumPa
     }
 
     /**
-     * Deploy BasicCoin smart contract
+     * Deploy ERC20 token smart contract
      * @return token smart contract object
      */
-    fun deployBasicCoinSmartContract(): BasicCoin {
+    fun deployERC20TokenSmartContract(): BasicCoin {
         return contract.BasicCoin.deploy(
             web3,
             credentials,
             gasPrice,
             gasLimit,
-            BigInteger.valueOf(1000),
+            BigInteger.valueOf(Long.MAX_VALUE),
```

Comment:
Do we really need so much tokens?

Reply:
Why not? For example, I want to deploy long term tokens in Ropsten for demo. 1000 is not enough definitely.